packages/ketcher-react/src/script/render/restruct/rerxnarrow.js

```diff
@@ -50,12 +51,17 @@ ReRxnArrow.prototype.makeSelectionPlate = function (restruct, paper, styles) {
 ReRxnArrow.prototype.show = function (restruct, id, options) {
   var render = restruct.render
   var centre = scale.obj2scaled(this.item.pp, options)
-  var path = draw.arrow(
-    render.paper,
-    new Vec2(centre.x - options.scale, centre.y),
-    new Vec2(centre.x + options.scale, centre.y),
-    options
-  )
+  var startPoint = new Vec2(centre.x - options.scale, centre.y)
+  var endPoint = new Vec2(centre.x + options.scale, centre.y)
+  var path
+  switch (this.item.mode) {
+    case RxnArrowMode.simple:
+      path = draw.arrow(render.paper, startPoint, endPoint, options)
+      break
+    case RxnArrowMode.equilibrium:
+      path = draw.equilibriumArrow(render.paper, startPoint, endPoint, options)
+      break
+  }
```

Comment:
What about default statement?

Reply:
We define the default value in the RxnArrow constructor: https://github.com/epam/ketcher/blob/4cbc1c26a23cb2a73408332f871d6e7caed8b3db/packages/ketcher-react/src/script/chem/struct/rxnArrow.ts#L22
Thus, all arrows will have the mode.